src/lib/consensus/proof_of_stake.ml

```diff
@@ -1936,7 +1889,7 @@ module Data = struct
         let%bind is_genesis = is_genesis next_global_slot in
         Boolean.Assert.any [global_slot_increased; is_genesis]
       in
-      let%bind next_epoch, next_slot =
+      let%bind next_epoch, _next_slot =
```

Comment:
Excuse my lack of knowledge regarding snarky internals, but does this create wasted constraints? Or does snarky only include constraints that contribute to the final checked computation?

Reply:
They're not wasted, we need to compute this value in order to compute divmod